### ISSUE TITLE
Update-Package does not support operations over a list of packages

### ISSUE THREAD

**Issue by rocifier** [NuGet/Home#7665]
Please read the following information before posting the issue.

## Details about Problem

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe):
VS2017

OS version (i.e. win10 v1607 (14393.321)):
win10

Worked before? If so, with which NuGet version:
never been implemented

## Detailed repro steps so we can see the same problem

For example, the command `Update-Package Package.A Package.B -IncludePrerelease` is not supported. I must run the Update-Package command once for every package I wish to update (in my use-case I don't want to update every package), but expect to be able to provide a list of packages to the command somehow. Updating all packages solution-wide or even project-wide is not an option for me.

Given the number of packages under a certain namespace combined with other networking-related nuget feed delays it takes me hours or even days to do this at work. Being able to drop in a list or a regexp filter of which packages I want to update means I could leave it running overnight or while I focus on other work. This often happens if making a single code change to one of our repositories that is responsible for building our business domain packages (I know... it needs separating "1 repository per nuget package", but there have never been strong guidelines on this and we learnt the hard way).

...

### Verbose Logs

```
PM> Update-Package Package.A Package.B -IncludePrerelease
Update-Package : Project 'Package.B' is not found.
```

**Comment by nkolev92** [comment]
Note that every package upgrade impacts the project graph. 
This means, that updating both Package A and B to the latest version **might** be impossible. 

One of the reasons why incremental updates make a lot of sense.

**Comment by rocifier** [comment]
Yes makes sense. It would need to operate at a higher level where it essentially runs the command for each package in order. Not sure how it would work transactionally if there was a failure in one of them part way through.

**Comment by nkolev92** [comment]
@rocifier 
Yeah the behavior would need to be defined in that case.
